# Table of Contents
 <p>

In [1]:
df = pd.read_csv("/Users/klay6683/Documents/planet4_docs/dimensions.csv", header=None)

In [2]:
df.head()

0    1    2
0  APF0000ib0  840  648
1  APF0000ib1  840  648
2  APF0000ib2  840  648
3  APF0000ib3  840  648
4  APF0000ib4  840  648

In [3]:
df.describe()

1              2
count  106344.0  106344.000000
mean      840.0     647.982322
std         0.0       0.826839
min       840.0     580.000000
25%       840.0     648.000000
50%       840.0     648.000000
75%       840.0     648.000000
max       840.0     648.000000

In [4]:
df[1].value_counts()

840    106344
Name: 1, dtype: int64

In [5]:
df[2].value_counts()

648    106294
612        46
604         2
580         2
Name: 2, dtype: int64

The fraction that is **not** 648 pixels high is $\frac{50}{106294}$ = {{50/106294}} (0.4 permille)